@here/harp-mapview/lib/Tile.ts

```diff
@@ -256,6 +256,11 @@ export interface TextElementIndex {
     elementIndex: number;
 }
 
+export type DynamicObjectUpdater = () => void;
```

Comment:
Please add a comment what a dynamic object is, i.e. it is one where the material needs to update at runtime, might be obvious for us, but not so obvious for new people starting the project.